config/dependency-injection/custom-loader.php

```diff
@@ -187,4 +191,15 @@ private function findClasses( $namespace, $pattern, $exclude ) {
 
 		return $classes;
 	}
+
+	/**
+	 * Normalize all slashes in a file path to forward slashes.
+	 *
+	 * @param string $path File path.
+	 *
+	 * @return string
```

Comment:
Very minor: it would be nice to add the returned value description. Will adjust during CR.